fibo.py

```diff
@@ -0,0 +1,5 @@
+def fibonacci_ruim(n):
+    if n <= 1:
+        return n
+    else:
+        return fibonacci_ruim(n-1) + fibonacci_ruim(n-2)
```

Comment:
This implementation of the Fibonacci sequence is inefficient due to its recursive nature without memoization, leading to a high computational cost for larger `n`. Consider using an iterative approach or adding memoization.
```suggestion
def fibonacci(n):
    a, b = 0, 1
    for _ in range(n):
        a, b = b, a + b
    return a
```